serpens/elastic.py

```diff
@@ -4,6 +4,10 @@
 from functools import wraps
 
 ELASTIC_APM_ENABLED = "ELASTIC_APM_SECRET_TOKEN" in os.environ
+ELASTIC_APM_CAPTURE_BODY = os.environ.get("ELASTIC_APM_CAPTURE_BODY")
+ELASTIC_APM_CAPTURE_RESPONSE_BODY = (
+    ELASTIC_APM_CAPTURE_BODY == "all" or ELASTIC_APM_CAPTURE_BODY == "transactions"
+)
```

Comment:
Acho melhor só transformar esta variável direto para verdadeiro/falso uma vez que ela tem um único propósito e acesso aqui.
```python
ELASTIC_APM_CAPTURE_BODY = os.getenv("ELASTIC_APM_CAPTURE_BODY") in ("all", "transactions")
```

Também acho melhor mover a definição destas variáveis todas que estão antes da importação do módulo `elastic` para depois e seguir a convenção de importação no início do módulo.